covalent_ui/webapp/.env.production

```diff
@@ -1 +1 @@
-REACT_APP_API_URL=
+PORT=49009
```

Comment:
will this force the front-end to run on this port?

Reply:
We will be removing this